src/components/BeerSearch/BeerSearch.js

```diff
@@ -0,0 +1,24 @@
+import React, {useState} from 'react';
+import axios from 'axios';
+import './BeerSearch.css'
+
+
+const BeerSearch = () => {
+    const [beerSearch, setBeerSearch] = useState([])
+
```

Comment:
To get your `handleSearchInput` function to work, you must use the search function you defined in `Beerlist.js` from the `props` and pass it `event.target.value`.
```
const {handleFilterSearch} = props;

    const handleSearchInput = (event) => {
        let initialState = event.target.value;
        handleFilterSearch(initialState)
    }